packages/@webex/plugin-meetings/src/members/index.ts

```diff
@@ -1018,4 +1018,39 @@ export default class Members extends StatelessWebexPlugin {
 
     return csis;
   }
+
+  /**
+   * Edit display name of participants in a meeting
+   * @param {string} memberId - id of the participant who is receiving request
+   * @param {string} requestingParticipantId - id of the participant who is sending request (optional)
+   * @param {string} [alias] - alias name
+   * @returns {Promise}
+   * @public
+   * @memberof Members
+   */
+  public editDisplayName(memberId: string, requestingParticipantId: string, alias: string) {
+    if (!this.locusUrl) {
+      return Promise.reject(
+        new ParameterError(
+          'The associated locus url for this meetings members object must be defined.'
+        )
+      );
+    }
+    if (!memberId) {
+      return Promise.reject(
+        new ParameterError('The member id must be defined to edit display name of the member.')
+      );
+    }
+
+    const {locusUrl} = this;
+
+    const options = MembersUtil.generateEditDisplayNameMemberOptions(
+      memberId,
```

Comment:
i think @InteractiveTimmy was referring to this, but instead of 4 params, can we instead pass an object with named params?

Reply:
I had just followed the actual existing conventions for the new created MembersUtil function 😀 This worked perfectly fine when I did the testing 👍🏻